lms/views/api/lti.py

```diff
@@ -18,13 +19,15 @@ class LTIOutcomesViews:
     def __init__(self, request):
         self.request = request
         self.parsed_params = self.request.parsed_params
-        self.lti_outcomes_client = self.request.find_service(name="lti_outcomes_client")
+        self.lti_grading_service: LTIGradingService = self.request.find_service(
```

Comment:
Not sure we've been doing this attribute typing thing but we have to import the service class anyway for `find_service`. 

With this I get auto-completion for self.lti_grading_service